dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ClientIT.java

```diff
@@ -231,8 +231,8 @@ public void pluginDownload() throws IOException {
      */
     @Test
     public void testDepsCommandWithVersionAndPython3() throws IOException {
-        Client.main(new String[] { "--config", TestUtility.getConfigFileLocation(true), "deps", "--client-version", "1.4.0", "--python-version", "3"});
-        Assert.assertTrue(systemOutRule.getLog().contains("avro-cwl=="));
+        Client.main(new String[] { "--config", TestUtility.getConfigFileLocation(true), "deps", "--client-version", "1.7.0", "--python-version", "3"});
+        Assert.assertFalse(systemOutRule.getLog().contains("monotonic=="));
```

Comment:
would be best if there is a still an `assertTrue` for something. 
i.e. currently, the test would pass if systemOutRule.getLog() was empty

Reply:
The line below this tests a bunch of true assertions